steam_auto_cracker_gui.py

```diff
@@ -84,8 +86,12 @@ def select_folder():
         selectedFolderLabel.config(text=f"Selected folder:\n{folder_path}")
         selectedFolderLabel.pack()
         frameGame2.pack()
+        gameNameEntry.delete(0, last=len(gameNameEntry.get()))
+        gameNameEntry.insert(0, folder_name)
         if gameSearchDone:
             frameCrack2.pack()
+            gameNameEntry.delete(0, last=len(gameNameEntry.get()))
+            gameNameEntry.insert(0, folder_name)

```

Comment:
Duplicate? Doesn't seem required

Reply:
Yes, you have two cases one of which when the folder already got selected. If the duplicate doesn‘t stay, it wouldn‘t auto-fill in that case

Reply:
Not true? It's not an if else, just a simple if. The delete & insert functions are called BEFORE the IF, so that wouldn't matter

Reply:
oh shoot true, yeah than nevermind probably

wouldn't the whole if clause be useless in this case?

Reply:
The IF close is meant to re-display the whole "Selected crack:" section if you decide to change the selected directory AFTER doing the App search (select a folder, search for the app, then unselect/change folder); at least from what I remember